cli/enry/main.go

```diff
@@ -139,33 +136,36 @@ func printPercents(out map[string][]string, buff *bytes.Buffer) {
 	}
 }
 
-func printFile(file string, buff *bytes.Buffer) {
+func printFileAnalysis(file string) string {
 	content, err := ioutil.ReadFile(file)
 	if err != nil {
 		fmt.Println(err)
 	}
-	totalLines, sloc := getLines(file, content)
+	totalLines, linesOfCode := getLines(file, string(content))
```

Comment:
Sorry to be picky here, but `linesOfCode` is just the same as `sloc` and equally ill-defined. As in my previous comment, unless a line of code has been unambiguously defined somewhere as non-blank lines for the whole project, it's just an arbitrary decision (the same way that in the text `sloc` should be just called `non-blank`)